mm2src/mm2_main/tests/docker_tests/qrc20_tests.rs

```diff
@@ -35,6 +35,7 @@ pub const QTUM_REGTEST_DOCKER_IMAGE: &str = "docker.io/sergeyboyko/qtumregtest";
+const TAKER_PAYMENT_SPEND_SEARCH_INTERVAL: f64 = 10.;
```

Comment:
I guess we can set this interval to 1s, what do you think?

Reply:
Yeah 1 second search interval for tests is a better idea